lib/constants.js

```diff
@@ -6,8 +6,9 @@ export const SUBS_NO_JOBS = SUBS.filter(s => s !== 'jobs')
 export const NOFOLLOW_LIMIT = 1000
 export const BOOST_MULT = 5000
 export const BOOST_MIN = BOOST_MULT * 5
-export const UPLOAD_SIZE_MAX = 2 * 1024 * 1024
+export const UPLOAD_SIZE_MAX = 10 * 1024 * 1024
```

Comment:
Increased size so camera upload works since most phones take higher quality pictures.

Also, maybe we can drop the upload size completely since we have fees now?

Reply:
We can set a different limit for avatars since avatars are free

Reply:
Dealer's choice.

On GrepMed, we added a bunch of limits because people would occasionally upload really odd images. The two that I remember were

1. huge image data sizes
2. normal image data size with massive image dimensions

Reply:
Using this now:

```
export const UPLOAD_SIZE_MAX = 25 * 1024 * 1024
export const UPLOAD_SIZE_MAX_AVATAR = 5 * 1024 * 1024
```